service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/returnedcase/ReturnedCaseTemplate.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.fpl.model.notify.returnedcase;
+
+import lombok.Getter;
+import lombok.Setter;
+import uk.gov.hmcts.reform.fpl.model.notify.NotifyData;
+
+@Getter
+@Setter
+public class ReturnedCaseTemplate implements NotifyData {
+    private String familyManCaseNumber;
+    private String returnedReasons;
+    private String returnedNote;
+    private String firstRespondentLastName;
+    private String firstRespondentFullName;
```

Comment:
Can these names be changed to just respondent last + full name? Don't think it matters from the template perspective as to what respondent data is being sent, just that some is